src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.test.tsx

```diff
@@ -62,6 +62,10 @@ import {
 } from './saved_objects_table';
 import { Flyout, Relationships } from './components';
 import { SavedObjectWithMetadata } from '../../types';
+import { WorkspaceObject } from 'opensearch-dashboards/public';
+import { DEFAULT_WORKSPACE_ID } from '../../../../../core/public';
+import { TableProps } from './components/table';
+import { waitFor } from '@testing-library/dom';
```

Comment:
These lines should be linted when commit(Absolute import should move before relative import), could you please take a look to see if the lint take effect when you do git commit?